### ISSUE TITLE
support additional formats

### ISSUE THREAD

**Issue by rikmeijer** [nostriphant/nip-19#1]
nprofile: a nostr profile
    nevent: a nostr event
    nrelay: a nostr relay
    naddr: a nostr replaceable event coordinate

**Comment by Sebastix** [comment]
nrelay is deprecated and the other identifiers are already in the package correct?

**Comment by rikmeijer** [comment]
Correct, I have not deprecated nrelay, one could still use it. The others (nprofile, naddr and nevent) are available
